.github/workflows/main.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [16]
+        node-version: [14, 16, 18]
```

Comment:
why add 14 here?

Reply:
I added 14 because its considered a [supported version](https://github.com/schultzp2020/roadie-backstage-plugins/blob/update-node/package.json#L6). I can remove 14 if you'd like.

Reply:
That makes sense now.. thanks.